go/vt/sqlparser/token.go

```diff
@@ -1027,11 +1031,59 @@ func (tkn *Tokenizer) scanMySQLSpecificComment() (int, []byte) {
 		}
 		tkn.consumeNext(buffer)
 	}
-	_, sql := ExtractMysqlComment(buffer.String())
-	tkn.specialComment = NewStringTokenizer(sql)
+
+	commentVersion, sql := ExtractMysqlComment(buffer.String())
+
+	mysqlVersion, err := convertMySQLVersionToCommentVersion(MySQLVersion)
+	if err != nil {
+		tkn.Error(err.Error())
+		return tkn.Scan()
+	}
```

Comment:
We can do this once at the start of VTGate as this value currently cannot change after VTGate startup.

Reply:
done